unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -90,13 +87,15 @@ public int numBytes(final byte b) {
    */
   public int length() {
```

Comment:
In Java string it is not ambiguous because people in general don't think about string in terms of a byte array